pyproject.toml

```diff
@@ -5,12 +5,12 @@
 requires = [
   "setuptools == 68.1.0",
   "setuptools_scm[toml] == 7.1.0",
-  "frequenz-repo-config[actor] == 0.8.0",
+  "frequenz-repo-config[lib] == 0.9.0",
```

Comment:
You could bump directly to v0.9.1 here, but dependabot will take care of it if you don't.